ext/drx/scatter_gather_shared.h

```diff
@@ -85,6 +85,7 @@ typedef struct _scatter_gather_info_t {
                      */
     bool scaled;
     bool is_scalar_value_signed;
+    bool is_replicating;
```

Comment:
Add comment about what some of these mean. To avoid the reader having to go to the manual.

Could you also add a link to the canonical reference to the manual anyway? Apologies if it's already there in the header and I missed it.

Reply:
Done.